API.md

```diff
@@ -1,17 +1,353 @@
 # Party Service API
 
-### The implemented API for this serivce can be found at /swagger/party/party-api.yaml
+### *Acceptance tests can be run to populate a local version with test data.*
 
-### CAVEAT: This page is subject to change while the Party service is being developed.
+---
 
-### Please contact the RAS development team before committing to these endpoints.
-  
-This page documents the Party service API endpoints. All endpoints return an `HTTP 200 OK` status code except where noted otherwise.
+## Business Endpoints:
 
-* `GET /parties/type/B/ref/499000011335`
-* `GET /parties/type/B/id/d826818e-179e-467b-9936-6a8603dc8b46`
+* `POST /party-api/v1/businesses`
+    * In the case of a new business, stores the new business data. 
+    * In the case of a known business then it adds the versioned attributes.
+    * Input data is validated against `party_schema.json` which mandates that `sampleUnitRef`, `sampleUnitType` and `sampleSummaryId` are mandatory.
+    * If attributes are provided then `runname1`,`runname2` and `runname3` are mandatory 
+### Example JSON Response
+```json
+{
+  "associations": [
+    
+  ],
+  "attributes": {
+     "birthdate": "1/1/2001",
+     "cell_no": 1,
+      "checkletter": "A",
+      "currency": "S",
+      "entname1": "Ent-1",
+      "entname2": "Ent-2",
+      "entname3": "Ent-3",
+      "entref": "Entref",
+      "entremkr": "Entremkr",
+     "formType": "FormType",
+     "formtype": "formtype",
+     "froempment": 8,
+     "frosic2007": "frosic2007",
+     "frosic92": "frosic92",
+     "frotover": 9,
+     "inclexcl": "inclexcl",
+     "legalstatus": "Legal Status",
+     "name": "Runame-1 Runame-2 Runame-3",
+     "region": "UK",
+     "runame1": "Runame-1",
+     "runame2": "Runame-2",
+     "runame3": "Runame-3",
+     "ruref": "831078087",
+     "rusic2007": "rusic2007",
+     "rusic92": "rusic92",
+     "seltype": "seltype",
+     "source": "test_get_business_by_search_partial_ru",
+     "trading_as": "Tradstyle-1 Tradstyle-2 Tradstyle-3",
+     "tradstyle1": "Tradstyle-1",
+     "tradstyle2": "Tradstyle-2",
+     "tradstyle3": "Tradstyle-3"
+    },
+  "id": "42e08c51-1d78-4524-bf39-c6523d4c0939",
+  "name": "Runame-1 Runame-2 Runame-3",
+  "sampleSummaryId": "831078087",
+  "sampleUnitRef": "831078087",
+  "sampleUnitType": "B",
+  "trading_as": "Tradstyle-1 Tradstyle-2 Tradstyle-3"
+}
+```
+---
+
+* `GET /party-api/v1/businesses?id=d826818e-179e-467b-9936-6a8603dc8b46&id=623435fa-708e-49c0-8f90-507ac862a540`
```

Comment:
don't include actual UUIDs except in the example. use square brackets for things which are optional and angular brackets for things which are mandatory, is a kind of convention

Reply:
Changed